contracts/src/token/erc721/mod.rs

```diff
@@ -1165,14 +1162,13 @@ mod tests {
     }
 
     #[motsu::test]
-    fn mints(contract: Erc721) {
+    fn when_minting_token_balance_and_owner_are_correct(contract: Erc721) {
```

Comment:
In this test we're also not checking for any error. Seems make sense to leave it as it is. cc @bidzyyys

Reply:
`mints` is fine here. The task is to rename tests that checks errors.